src/modal/modal.ts

```diff
@@ -6,10 +6,21 @@ export class Modal extends LitElement implements HTMLDialogElement {
     ...LitElement.shadowRootOptions,
     delegatesFocus: true,
   };
-
   @property({ type: Boolean, reflect: true })
   open: boolean = false;
 
+  @property({ type: String })
+  title: string = "";
+
+  @property({ type: String })
+  description: string = "";
+
+  @property({ type: String })
+  alignment: string = 'right';
```

Comment:
Can we make the type of this property more specific using union operation? for example `'left' | 'center' | 'right'`